buildSrc/src/main/java/org/opensearch/gradle/info/GlobalBuildInfoPlugin.java

```diff
@@ -112,6 +113,8 @@ public void apply(Project project) {
         BuildParams.init(params -> {
             // Initialize global build parameters
             boolean isInternal = GlobalBuildInfoPlugin.class.getResource("/buildSrc.marker") != null;
+            var cryptoStandard = System.getenv(OPENSEARCH_CRYPTO_STANDARD);
```

Comment:
I think it would be useful to extract this into `FipsBuild` helper class (or similar), so we could consolidate `OPENSEARCH_CRYPTO_STANDARD` usage in 1 place:

```
public final class FipsBuildParams {
    private static final String fipsMode = System.getenv("OPENSEARCH_CRYPTO_STANDARD");
    
    private FipsBuildParams() {
    }
    
    public static boolean isInFipsMode() {
        return "FIPS-140-3".equals(fipsMode);
    }
    
    public static String getFipsMode() {
        return fipsMode;
    }
}

```